carma-messenger-config/development/docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
     command: bash -c 'wait-for-it localhost:11311 -- rosparam load /opt/carma/vehicle/config/bridge.yml && source ~/.base-image/workspace/install/setup.bash && ros2 run ros1_bridge dynamic_bridge --multi-threads'
 
   messenger:
```

Comment:
same comment as before where we probably need to switch

Reply:
updated